cmd/lncli/commands.go

```diff
@@ -1133,9 +1140,10 @@ var addInvoiceCommand = cli.Command{
 	Usage: "add a new invoice.",
 	Description: `
 	Add a new invoice, expressing intent for a future payment.
-	
-	The value of the invoice in satoshis is necessary for the creation, 
-	the remaining parameters are optional.`,
+
+	Invoices without an amount can be created by not supplying any
+	parameters or providing an amount of 0. These invoices allow the payee
+	to specify the amount of satoshis they wish to send.`,
```

Comment:
The `payinvoice` proxy command should be updated as well, since that's what most users default to these days.